localization/yabloc/yabloc_common/src/ublox_stamp.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "yabloc_common/ublox_stamp.hpp"
+
+#include <time.h>
+
+namespace yabloc::common
+{
+rclcpp::Time ublox_time_to_stamp(const ublox_msgs::msg::NavPVT & msg)
+{
+  // TODO: Day 31 may cause invalid convertion
```

Comment:
How about writing a test for this? (Which should not take so much time if we use ChatGPT)

Reply:
I will remove this file because all nodes use `/sensing/gnss/pose` rather than `ublox_msgs` now.

Reply:
I removed all files related to ublox in 0b15a74 & e7a0dc4 .